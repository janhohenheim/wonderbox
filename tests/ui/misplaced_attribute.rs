use wonderbox::autoresolvable;

#[autoresolvable]
struct Struct {}

#[autoresolvable]
trait Trait {}

#[autoresolvable]
fn function() {}

fn main() {}
