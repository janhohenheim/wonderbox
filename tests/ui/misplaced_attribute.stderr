error: #[autoresolvable] needs to be placed over an impl block
 --> $DIR/misplased_attribute.rs:4:1
  |
4 | struct Struct {}
  | ^^^^^^

error: #[autoresolvable] needs to be placed over an impl block
 --> $DIR/misplased_attribute.rs:7:1
  |
7 | trait Trait {}
  | ^^^^^

error: #[autoresolvable] needs to be placed over an impl block
  --> $DIR/misplased_attribute.rs:10:1
   |
10 | fn function() {}
   | ^^
